src/dispensary.js

```diff
@@ -65,13 +67,36 @@ export default class Dispensary {
       });
   }
 
-  getLibraries() {
+  // Matches only against cached hashes; this is the API external apps and
+  // libraries would use.
+  match(contents, _hashesFile=DEFAULT_HAHES_FILE, _cachedHashes=null) {
+    if (this._cachedHashes === null) {
```

Comment:
Fair enough. It's preventing fs activity if we've already loaded the hash file, but really that should be abstracted away from this function. I'll clean it up.